### PR TITLE
0303 update version

### PR DESCRIPTION
**Pull request by wivwiv**

### CODE REVIEW

File: zh_CN/deploy/install-centos.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ::: tab amd64
 
-1. 下载 [emqx-5.0.17-el8-amd64.rpm](https://www.emqx.com/zh/downloads/broker/5.0.17/emqx-5.0.17-el8-amd64.rpm)https://www.emqx.com/downloads/broker/v5.0.14/emqx-5.0.14-el8-amd64.rpm)。
+1. 下载 [emqx-5.0.17-el8-amd64.rpm](https://www.emqx.com/zh/downloads/broker/5.0.17/emqx-5.0.17-el8-amd64.rpm)https://www.emqx.com/downloads/broker/v5.0.19/emqx-5.0.19-el8-amd64.rpm)。
```
Comment (LenaLenaPan) [inline zh_CN/deploy/install-centos.md]: 这里的版本不对

**Review by LenaLenaPan — APPROVED** [review]

File: en_US/deploy/install-macOS.md [hunk 2]
```diff
@@ -7,20 +7,20 @@ Supported versions:
 - macOS 12
 - macOS 11
 
-The section below will take macOS 12 as an example to illustrate how to download the latest version of EMQX. For other If you want to install a different version or in a different system, please visit the [EMQX Deployment page](https://www.emqx.com/zh/try?product=enterprise). 
+The section below will take macOS 12 as an example to illustrate how to download the latest version of EMQX. For other If you want to install a different version or in a different system, please visit the [EMQX Deployment page](https://www.emqx.com/en/try?product=enterprise). 
 
 {% emqxce %}
 
-1. Download [emqx-5.0.17-macos12-arm64.zip](https://www.emqx.com/en/downloads/broker/5.0.17/emqx-5.0.17-macos12-arm64.zip). 
+1. Download [emqx-5.0.19-macos12-arm64.zip](https://www.emqx.com/en/downloads/broker/5.0.19/emqx-5.0.19-macos12-arm64.zip).
```
Comment (id) [inline en_US/deploy/install-macOS.md:14]: add an option to use [Homebrew](https://brew.sh/)?

```
brew install emqx
```

Comment (id) [reply en_US/deploy/install-macOS.md:14]: Also, if you add a link to arm64 build, probably a good idea to mention that this package is for Apple Silicon macs

Comment (id) [reply en_US/deploy/install-macOS.md:14]: macos12 amd64 binary package will be available starting from v5.0.20

Comment (wivwiv) [reply en_US/deploy/install-macOS.md:14]: @LenaLenaPan you can add it to the task list.

**Pull request merged**